packages/suite/src/components/suite/images/DeviceConfirmImage.tsx

```diff
@@ -4,24 +4,18 @@ import { Image, ImageProps } from '@trezor/components';
 import { ImageType } from '@trezor/components/src/components/Image/Image';
 import { getDeviceModel } from '@trezor/device-utils';
 
-const getImage = (majorVersion: string): ImageType => {
-    switch (majorVersion) {
-        case '1':
-            return 'ONE_DEVICE_CONFIRM';
-        case 'T':
-        default:
-            return 'T_DEVICE_CONFIRM';
-    }
-};
-
 interface DeviceConfirmImageProps extends Omit<ImageProps, 'image'> {
     device: TrezorDevice;
 }
 
 export const DeviceConfirmImage = ({ device, ...rest }: DeviceConfirmImageProps) => {
-    const majorVersion = getDeviceModel(device);
+    const deviceModel = getDeviceModel(device);
+
+    if (!deviceModel) {
+        return null;
+    }
 
-    const imgName = getImage(majorVersion);
+    const imgName = `${deviceModel}_DEVICE_CONFIRM` as ImageType;
```

Comment:
If you use `as const`, typescript build the type correctly
```suggestion
    const imgName = `${deviceModel}_DEVICE_CONFIRM` as const;
```

Reply:
a6bbc8ed0df8fab2e04fb0a86ad46ec354ead91b